models/users.js

```diff
@@ -483,6 +483,28 @@ const getDiscordUsers = async () => {
   }
 };
 
+const updateRoles = async (userData, newRoles) => {
+  try {
+    const roles = { ...userData.roles };
+    const newRolesArray = Object.entries(newRoles);
+    const rolesContainNewRoles = newRolesArray.every(([key, value]) => {
+      return roles[String(key)] === value;
+    });
```

Comment:
did adding `String` keyword solved `Generic Object Injection Sink` warning. ?

Reply:
Yes because attacker could  provide something unexpected may be integer etc. and this turn into  unpredictable behaviour